code/18-switch-mass-hypo/ntuple_switchHypo.py

```diff
@@ -0,0 +1,69 @@
+from GaudiConf import IOHelper
+from Configurables import DaVinci, DecayTreeTuple
+from DecayTreeTuple.Configuration import *
+
+# Stream and stripping line we want to use
+stream = 'AllStreams'
+line = 'D2hhCompleteEventPromptDst2D2RSLine'
+tesLoc = '/Event/{0}/Phys/{1}/Particles'.format(stream, line)
+
+# prepare a Filter algorithm to creat a new Selection of particles
+from Configurables import FilterDesktop
+alg = FilterDesktop('SelFilter')
```

Comment:
I am not convinced that we need this inital Filter here.
Let me try to simplify this.